content/en/docs/contribute/participate/roles-and-responsibilities.md

```diff
@@ -147,7 +147,7 @@ separately for reviewer status in SIG Docs.
 To apply:
 
 1. Open a pull request that adds your GitHub user name to a section of the
-   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS) file
+   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES) file
```

Comment:
@bradtopol I investigated these links (also line 222) and the changes look good to me

Reply:
Great catch!